tests/php/src/Helpers/MockAdminUser.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Trait MockAdminUser.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Tests\Helpers;
+
+/**
+ * Helper trait for mocking an user:
+ * - Admin user in single site.
+ * - Super admin user in multisite.
+ *
+ * @package AmpProject\AmpWP
+ */
+trait MockAdminUser {
+
+	/**
+	 * Mock an admin or super admin user.
+	 *
+	 * @return WP_User
+	 */
+	public function mock_admin_user() {
+		$admin_user = self::factory()->user->create_and_get( [ 'role' => 'administrator' ] );
+
+		if ( is_multisite() ) {
+			$admin_user = self::factory()->user->create_and_get(
+				[
+					'role' => 'administrator',
+				]
+			);
+
```

Comment:
This user object seems to be created as a duplicate.

```suggestion
```

Reply:
Ahh. yes.